### ISSUE TITLE
Invalid warning

### ISSUE THREAD

**Issue by kobezzza** [rossPatton/stylint#244]
`colons: "never"`

```css
input {
  &::-ms-clear // Warning
	display none
}
```
